fineract-provider/src/main/java/org/apache/fineract/cob/api/ConfigureBusinessStepResource.java

```diff
@@ -69,6 +71,20 @@ public class ConfigureBusinessStepResource {
     private final ConfigJobParameterService configJobParameterService;
     private final PortfolioCommandSourceWritePlatformService commandWritePlatformService;
 
+    @GET
+    @Path("/names")
+    @Produces({ MediaType.APPLICATION_JSON })
+    @Operation(summary = "List Business Jobs", description = "Returns the configured Business Jobs")
```

Comment:
I'm not sure what you mean by "Business Jobs". Probably you wanna write something like "List of jobs with configurable business steps".

Reply:
This endpoint is for getting the Workflow Job Name (Previously Business Jobs), Because currently there is only one, LOAN_CLOSE_OF_BUSINESS but It's possible to have more, and base on the Workflow Job selected in the dropdown you'll be able to see the different Steps and change the order